server/routes/createGoal/createGoalsController.ts

```diff
@@ -12,44 +12,37 @@ export default class CreateGoalsController {
     const { prisonNumber } = req.params
     const { prisonerSummary } = req.session
 
-    req.session.createGoalsForm = req.session.createGoalsForm || []
-
-    if (!req.session.createGoalsForm) {
-      req.session.createGoalsForm = {
-        prisonNumber,
-        goals: [
-          {
-            steps: [{ title: '' }],
-          },
-        ],
-      } as CreateGoalsForm
+    const createGoalsForm = req.session.createGoalsForm || {
+      prisonNumber,
+      goals: [
+        {
+          steps: [],
+        },
+      ],
     }
+    req.session.createGoalsForm = undefined
 
-    const today = moment().toDate()
+    const today = startOfDay(new UTCDate())
```

Comment:
```suggestion
    const today = startOfToday()
```

Reply:
date-fns has a `startOfToday` method you could use here instead, https://date-fns.org/v3.6.0/docs/startOfToday

Reply:
Done 👍